handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1237,13 +1235,7 @@ private String toJavaCipherSuite(String openSslCipherSuite) {
         }
 
         String prefix = toJavaCipherSuitePrefix(SSL.getVersion(ssl));
-        String converted = CipherSuiteConverter.toJava(openSslCipherSuite, prefix);
-        if (converted != null) {
-            openSslCipherSuite = converted;
-        } else {
-            openSslCipherSuite = prefix + '_' + openSslCipherSuite.replace('-', '_');
-        }
-        return openSslCipherSuite;
+        return CipherSuiteConverter.toJava(openSslCipherSuite, prefix);
```

Comment:
Correct.  This was a warning that sonarqube picked up on.  That is why I added the comments here `This will not be {@code null}.` https://github.com/netty/netty/pull/3366/files#diff-3c65ae995e11464b5efbd26727272e85R273.